vbox_create.sh

```diff
@@ -177,6 +167,15 @@ function create_cluster_VMs {
   VBN2="${bootstrap_interfaces[2]?Need a Virtualbox network 3 for the bootstrap}"
 
   #
+  # Delete any old (or other clusters') IPXE disks to avoid collisions
+  #
+  old_ipxe=$($VBM list hdds | egrep '^Location:.*ipxe.vdi' | \
+             sed 's/Location:[ ]*//')
+  [ -n "$old_ipxe" ] && for disk in $old_ipxe; do
+    $VBM closemedium disk $disk --delete
```

Comment:
Does this work when the other cluster is still running? I get errors from virtualbox that it is locked:

```
$ vboxmanage closemedium disk ~/chef-bcpc/vbox/ipxe.vdi --delete
VBoxManage: error: Medium '~/chef-bcpc/vbox/ipxe.vdi' is locked for reading by another task
VBoxManage: error: Details: code VBOX_E_INVALID_OBJECT_STATE (0x80bb0007), component MediumWrap, interface IMedium, callee nsISupports
VBoxManage: error: Context: "DeleteStorage(pProgress.asOutParam())" at line 1602 of file VBoxManageDisk.cpp
VBoxManage: error: Failed to delete medium. Error code Unknown Status -2135228409 (0x80bb0007)
VBoxManage: error: Medium '~/chef-bcpc/vbox/ipxe.vdi' is locked for reading by another task
VBoxManage: error: Details: code VBOX_E_INVALID_OBJECT_STATE (0x80bb0007), component MediumWrap, interface IMedium, callee nsISupports
VBoxManage: error: Context: "Close()" at line 1611 of file VBoxManageDisk.cpp

```